src/plugins/discover/public/application/components/default_discover_table/_doc_table.scss

```diff
@@ -128,6 +128,13 @@ doc-table {
 }
 
 .osd-table {
+  thead {
+    position: sticky;
+    top: 0;
+    background-color: white;
```

Comment:
what if toggle in darkmode

also in general we should sass variables so that if there is a color scheme change then holistically it will change if it uses the sass variables

Reply:
ah good comment thanks @kavilla . something to keep in mind that iw asn't thinking about